### ISSUE TITLE
[1.0.0] Vector Manip Levelup takes exceedingly long.

### ISSUE THREAD

**Issue by Renniuq** [LambdaInnovation/AcademyCraft#318]
I've just gotten Vector Manipulation on a world, and I have managed to max Directed Shock and 10% Groundshock, and yet I'm only 15% of the way to level 2. I'm not sure, but I feel that this is taking long enough that it's probably a bug. If its intended, then I guess just close this.

**Comment by WeAthFoLD** [comment]
Yea I've already seen lots of people complaining this problem. Considering tweaking the data a little bit.

**Comment by Renniuq** [comment]
Ah, ok. Just thought I'd point it out in case it wasn't intended.

**Comment by EndermanParty** [comment]
To me this seems to be the case with every class other than the meltdowner, as the meltdowner is the only class where the first 4 abilities are offensive and typically used quite often.

**Comment by Renniuq** [comment]
It might simply be that I'm in a skyblock, and as such can't often run into situations where I fight monsters. Since I either get no monsters, or craptons of them. and the whole "break-the-ground-and-hit-the-monsters" doesn't sound very appealing on a one block thick platform over the void, where making said hole will release a torrent of monsters upon me. I guess I'll just have to go 0.1% at a time in groundshock. I've used it 250 times so far! 750 to go. And at the current rate, I'll be just over 35% of the way to level 2 by that point... Oh dear...

**Comment by WeAthFoLD** [comment]
@Renniuq That makes sense. Currently I'm afraid that the only workaround would be turning off the block destruction in this kind of situation. Anyways, in (very recent) next iterations, we will go through tweaking data for categories & skills, and the skill data customization feature will be finished, so players will be able to fine-tune the grinding speed as they see fit.

**Comment by Renniuq** [comment]
Oh, I love the idea of fine-tuning the speed. That will more than make up for the problems I'm having here. I enjoy high-levels of customization. I have a nice little spawner that will funnel all the mobs into a 1x2 space with a half block gap I can punch through, so that's why I easily upgraded the Directed Shock, but I tried the Groundshock on that, and it broke the thing and monsters started pouring out. I have been smashing my newly 2-block thick platform ever since, and just building the stuff back. I've managed to get to just over 50% on Groundshock, and am now 38% of the way through level 1, so tweaking for a skyblock is most definitely a must. 0.1% experience per use of an ability that fills my overload bar (gives 120 overload, of my max of 160 due to the extra 40 from using... interestingly, that maxed out way faster than the CP cap bonus. I use the debug screen a fair bit.) does make for a hell of a grind. I've taken to just using the fullcp command to simply speed things up, since at this point I'm doing nothing else but leveling this ability, and it simply ends up saving time. I have noticed it only fills to the default max cp of 1800 and doesn't account for my extra cp from using abilities. I can make a different report for that, but it seems really minor and not something many people would be upset about.
